pynotify/models.py

```diff
@@ -9,7 +9,7 @@
 from django.template.base import Context, Template
 from django.utils.functional import cached_property
 from django.utils.translation import ugettext as _
-from django.utils.translation import ugettext_lazy as _l
+from django.utils.translation import ugettext_lazy
```

Comment:
To fix problem with generating migrations, I had to replace `_l` -> `ugettext_lazy`, since standard `makemigrations` does not recognize `_l` (I couldn't figure out how to run the chamber command, which does recognize this)